client/index.tsx

```diff
@@ -44,7 +44,12 @@ const unsubscribe = window.storeRedux.subscribe(() => {
     window.reduxState = window.storeRedux.getState()
 })
 
-window.socketIoClient = io()
+const { pathname } = window.location
+const socketServerPath =
+    pathname + (pathname.endsWith('/') ? '' : '/') + 'socket.io/'

```

Comment:
Won't this only work if the pathname is exactly the `ROOT_PATH`?

Reply:
I think that's right, but does it matter if the client is an SPA always exposed from `/`?

Reply:
For Sysifos it's only ever served from `/` as there's no React Router right?
So I'm not sure it would ever be able to be on a different path unless you were doing weird routing via a reverse proxy?

Reply:
I don't think it matters for the versions of Sisyfos, since Sisyfos the only route is root. It is just something to be aware of, if Sisyfos shoulb to be extended with in-app routing in the future.